src/e2e/java/teammates/e2e/cases/lnp/FeedbackSessionViewLNPTest.java

```diff
@@ -0,0 +1,244 @@
+                FeedbackQuestionDetails details = new FeedbackTextQuestionDetails("Test Question");
+                feedbackQuestions.put("QuestionTest",
+                        FeedbackQuestionAttributes.builder()
+                            .withFeedbackSessionName("Test Feedback Session")
+                            .withQuestionDescription("Test Question")
+                            .withCourseId(COURSE_ID)
+                            .withQuestionDetails(details)
+                            .withQuestionNumber(1)
+                            .withGiverType(FeedbackParticipantType.SELF)
+                            .withRecipientType(FeedbackParticipantType.NONE)
+                            .withShowResponsesTo(showResponses)
+                            .withShowGiverNameTo(showGiverName)
+                            .withShowRecipientNameTo(showRecepientName)
+                            .build()
+                );
+                return feedbackQuestions;
+            }
+
+            @Override
+            public List<String> generateCsvHeaders() {
+                List<String> headers = new ArrayList<>();
+
+                headers.add("loginId");
+                headers.add("isAdmin");
+                headers.add("googleId");
+                headers.add("courseId");
+                headers.add("fsname");
+
+                return headers;
+            }
+
+            @Override
+            public List<List<String>> generateCsvData() {
+                DataBundle dataBundle = loadDataBundle(getJsonDataPath());
+                List<List<String>> csvData = new ArrayList<>();
+
+                dataBundle.students.forEach((key, student) -> {
+                    List<String> csvRow = new ArrayList<>();
+
+                    csvRow.add(student.googleId); // "googleId" is used for logging in, not "email"
+                    csvRow.add("no");
+                    csvRow.add(student.googleId);
+                    csvRow.add(COURSE_ID);
+                    csvRow.add("Test Feedback Session");
+
+                    csvData.add(csvRow);
+                });
+
+                return csvData;
+            }
+        };
+    }
+
+    @Override
+    protected ListedHashTree getLnpTestPlan() {
+        ListedHashTree testPlan = new ListedHashTree(JMeterElements.testPlan());
+        HashTree threadGroup = testPlan.add(
+                JMeterElements.threadGroup(NUMBER_OF_USER_ACCOUNTS, RAMP_UP_PERIOD, 1));
+        threadGroup.add(JMeterElements.csvDataSet(getPathToTestDataFile(getCsvConfigPath())));
+        threadGroup.add(JMeterElements.cookieManager());
+        threadGroup.add(JMeterElements.defaultSampler());
+        threadGroup.add(JMeterElements.onceOnlyController())
+                .add(JMeterElements.loginSampler());
+
+        // Add HTTP samplers for test endpoint
+        String firstPath = "webapi/student?courseid=${courseId}";
```

Comment:
Is there any particular reason to name the path as `firstPath`, `secondPath` and so on?

Reply:
No particular reason at first, it was when I was trying stuff out. I changed the name to be more explanatory of what it does.